specs/latest/1.0/index.html

```diff
@@ -3590,7 +3589,11 @@ <h3><a name="ATTRIBS_AND_RANGE_CHECKING">Enabled Vertex Attributes and Range Che
 cause any error to be generated during a call to <code>drawArrays</code>
 or <code>drawElements</code>.
 
-</p>
+    <h3>Out-of-bounds fetches from the index buffer</h3>
+
+<p>
+Calling an indexed drawing command (<code>drawElements</code>) that fetches index elements outside
+the bounds of ELEMENT_ARRAY_BUFFER will result in an INVALID_OPERATION error.
 
```

Comment:
I presume we'll need to modify @null77 's [Don't no-op draw calls for zero count in validation](https://github.com/google/angle/commit/9fdaa497b489b7e7739e29812c85da2125b6a681) change to take these new rules into account?

Reply:
Yes, probably. Firefox does this validation before we call into ANGLE, fwiw.